docs/source-fabric/api/accelerators.rst

```diff
@@ -19,4 +19,4 @@ Accelerators
     CPUAccelerator
     CUDAAccelerator
     MPSAccelerator
-    TPUAccelerator
+    XLAAccelerator
```

Comment:
The renames of the plugins LGTM. I would not rename the accelerator though, since TPU is the accelerator and what we support right now. Can we please keep it?

Reply:
With https://github.com/Lightning-AI/lightning/issues/16130, there will be code in the XLAAcelerator like

```python
from torch_xla.experimental import tpu, gpu

if gpu.is_available()
   # XLA gpu support
else:
   # XLA tpu support
```

Making `TPUAccelerator` incorrect as a name